lib/internal/modules/esm/fetch_module.js

```diff
@@ -0,0 +1,285 @@
+'use strict';
+const {
+  globalThis,
+  ArrayPrototypePush,
+  Promise,
+  PromisePrototypeThen,
+  PromiseResolve,
+  SafeMap,
+  StringPrototypeEndsWith,
+  StringPrototypeSlice,
+  StringPrototypeStartsWith,
+} = primordials;
+const {
+  Buffer: {
+    concat: BufferConcat
+  }
+} = globalThis;
```

Comment:
This should import from `require('buffer')` rather than `globalThis`

Reply:
Thanks. While we’re here, I would appreciate if https://github.com/nodejs/node/pull/38635 could get unblocked and land.